application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -3881,6 +3883,18 @@ public function cpd_importParticipants($sSessionKey, $participants, $update = fa
             }
         }
 
+        // Remove param is set to True
```

Comment:
All of this can be replaced by

```
Can be replaced by 
$participantsId = array_column($participants, 'participant_id');
$criteria = new CDbCriteria();
$criteria->addNoInCondition('participant_id', $participantsId);
$aResponse['RemoveCount'] = Participant::model()->deleteAll($criteria);
```

Reply:
Thanks! 
addNoInCondition => addNo**t**InCondition

Reply:
Yes ;)